packages/utils/src/credential-utils.ts

```diff
@@ -12,6 +13,7 @@ import {
 import { blake2bHex } from 'blakejs'
 import { decodeJWT } from 'did-jwt'
 import { normalizeCredential, normalizePresentation } from 'did-jwt-vc'
+import { CredentialStatus } from 'credential-status'
```

Comment:
This is no longer needed, since the `isRevoked` method is only used in the `credential-w3c` package
```suggestion
```